code/modules/mob/living/simple_animal/hostile/megafauna/hierophant.dm

```diff
@@ -430,20 +430,19 @@ Difficulty: Hard
 		set_stat(CONSCIOUS) // deathgasp won't run if dead, stupid
 		..(force_grant = stored_nearby)
 
-/mob/living/simple_animal/hostile/megafauna/hierophant/devour(mob/living/L)
-	for(var/obj/item/W in L)
-		if(!L.dropItemToGround(W))
-			qdel(W)
+/mob/living/simple_animal/hostile/megafauna/hierophant/celebrate_kill(mob/living/L)
 	visible_message(span_hierophant_warning("\"[pick(kill_phrases)]\""))
-	visible_message(span_hierophant_warning("[src] annihilates [L]!"),span_userdanger("You annihilate [L], restoring your health!"))
-	adjustHealth(-L.maxHealth*0.5)
-	L.investigate_log("has been devoured by [src].", INVESTIGATE_DEATHS)
-	L.dust()
+	visible_message(span_hierophant_warning("[src] annihilates [L]'s organs!"),span_userdanger("You annihilate [L]'s organs, restoring your health!"))
```

Comment:
```suggestion
	visible_message(span_hierophant_warning("[src] obliterates [L]!"),span_userdanger("You absorb [L]'s life force, restoring your health!"))
```
Same here wrt "I just think this message looks silly" but you can feel free to disagree on that.